components/assetcard/index.css

```diff
@@ -227,37 +233,38 @@ governing permissions and limitations under the License.
   background-color: transparent;
 
   pointer-events: none;
-  transition: background-color var(--spectrum-global-animation-duration-100) ease-in-out;
+  transition: background-color var(--mod-assetcard-asset-animation-duration, var(--spectrum-assetcard-asset-animation-duration)) ease-in-out;
 
-  border-radius: calc(var(--spectrum-assetcard-border-radius) - 1px);
+  border-radius: calc(var(--mod-assetcard-border-radius, var(--spectrum-assetcard-border-radius)) - 1px);
 }
 
 /* the checkbox or selection order indicator */
 .spectrum-AssetCard-selectionIndicator {
   position: absolute;
   z-index: 2;
-  inset-inline-start: var(--spectrum-assetcard-selectionindicator-margin);
-  inset-block-start: var(--spectrum-assetcard-selectionindicator-margin);
+  inset-inline-start: var(--mod-assetcard-selectionindicator-margin, var(--spectrum-assetcard-selectionindicator-margin));
+  inset-block-start: var(--mod-assetcard-selectionindicator-margin, var(--spectrum-assetcard-selectionindicator-margin));
 
   display: flex;
   align-items: center;
   justify-content: center;
 
-  inline-size: var(--spectrum-assetcard-selectionindicator-size);
-  block-size: var(--spectrum-assetcard-selectionindicator-size);
+  inline-size: var(--mod-assetcard-selectionindicator-size, var(--spectrum-assetcard-selectionindicator-size));
+  block-size: var(--mod-assetcard-selectionindicator-size, var(--spectrum-assetcard-selectionindicator-size));
 
-  border-radius: var(--spectrum-assetcard-selectionindicator-border-radius);
+  border-radius: var(--mod-assetcard-selectionindicator-border-radius, var(--spectrum-assetcard-selectionindicator-border-radius));
 
-  box-shadow: 0 var(--spectrum-global-dimension-static-size-25) var(--spectrum-global-dimension-static-size-75) rgb(0, 0, 0, 0.15);
+  box-shadow: 0 var(--spectrum-assetcard-selectionindicator-offset-y) var(--spectrum-assetcard-selectionindicator-blur) rgb(0, 0, 0, 0.15);
 
-  color: var(--spectrum-global-color-static-white);
-  font-weight: var(--spectrum-global-font-weight-bold);
-  font-size: var(--spectrum-global-dimension-font-size-400);
+  color: var(--highcontrast-assetcard-selectionindicator-color, var(--mod-assetcard-selectionindicator-color, var(--spectrum-assetcard-selectionindicator-color)));
+  font-weight: var(--mod-assetcard-selectionindicator-font-weight, var(--spectrum-assetcard-selectionindicator-font-weight));
+  font-size: var(--mod-assetcard-selectionindicator-font-size, var(--spectrum-assetcard-selectionindicator-font-size));
 
-  visibility: hidden;
+  display: none;
```

Comment:
Is this overwriting the display: flex above on 248?

Reply:
yes it is, I'll remove the `display: flex` above this